resources/datadog-integrations/project.rb.erb

```diff
@@ -43,8 +43,11 @@ end
 dependency 'preparation'  # creates required build directories
 dependency 'dd-check-<%= name %>-software'
 
+# Checks should require the datadog agent
+runtime_dependency 'datadog-agent'
```

Comment:
Does it play nicely with both debs and rpms ? (I guess yes, just want to confirm)